include/evmc/evmc.h

```diff
@@ -722,6 +782,13 @@ struct evmc_instance
     /** Pointer to function executing a code by the EVM instance. */
     evmc_execute_fn execute;
 
+    /**
+     * Optional pointer to function setting the EVM instruction tracer.
+     *
+     * If the EVM does not support this feature the pointer can be NULL.
+     */
+    evmc_set_tracer_fn set_tracer;
```

Comment:
Can this be unset after its being set?

I don't fully remember (and seemingly our documentation isn't clear enough): can an `evmc_instance` reused for multiple executions?

Reply:
Good question. Answered in the latest commit.

Reply:
That commit looks good.